src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-installation/nuxeo-retention-installation-standard.md

```diff
@@ -27,6 +27,16 @@ This page gives all the necessary steps to install the Retention Management addo
 
 ## Configuration
 
+### Setting the Addon in Standard Mode
+Setting the addon in standard mode is done through the following [nuxeo.conf]({{page page='configuration-parameters-index-nuxeoconf'}}) property:
+```
+nuxeo.retention.strictmode.enabled=false
+```
+
+{{#> callout type='info'}}
+Since LTS 2021 HF41, this property replaces and supersedes the property `nuxeo.retention.compliance.enabled`: if both are set, the value of nuxeo.retention.strictmode.enabled will be used.
```

Comment:
```suggestion
Since LTS 2021 HF41, this property replaces and supersedes the property `nuxeo.retention.compliance.enabled`: if both are set, the value of `nuxeo.retention.strictmode.enabled` will be used.
```